gradle.properties

```diff
@@ -1,5 +1,5 @@
 # Kompendium
-project.version=4.0.0
+project.version=4.1.0
```

Comment:
wasn't sure what version to update here.

Reply:
Will still be 4.0.0 when it get's released, since there has been no 4.0.0 release yet :)